scripts/dev_setup.py

```diff
@@ -25,13 +25,15 @@ def exec_command(command):
 print('Running dev setup...')
 print('Root directory \'{}\'\n'.format(root_dir))
 
+# install general requirements
+exec_command('pip install -r requirements.txt')
```

Comment:
I think we no longer have the problem because we now depend on the individual packages. IIRC, the problem originated because we had modules depending on the roll-up package, which would replace the version of KV management that it needed with an older version.